tests/forte/data/multi_pack_test.py

```diff
@@ -198,6 +203,196 @@ def test_multipack_entries(self):
             ],
         )
 
+    def test_remove_pack(self):
+
+        """
+        1. Test to remove the added pack from multi_pack which is independent;
+        2. Test to remove the added pack from multi_pack with MultiPackGroup and MultiPackLink;
+
+        Returns:
+
+        """
+
+        self.data_pack10 = self.multi_pack.add_pack(ref_name="remove pack 10")
```

Comment:
Most of these variables should be used locally in this function, so there is no need to do `self.data_pack10`, just store it to `data_pack10`